src/js/components/Page.js

```diff
@@ -179,6 +180,23 @@ var Page = React.createClass({
     );
   },
 
+  getLicensingBanner() {
```

Comment:
As pointed out in https://github.com/mesosphere/dcos-ui-plugins-private/pull/636#pullrequestreview-70143342 I think we usually try avoiding such words in the open version. Name suggestion `getTopBanner()`